modules/tinymce/src/core/main/ts/content/InsertContentImpl.ts

```diff
@@ -22,6 +22,19 @@ import * as SelectionUtils from '../selection/SelectionUtils';
 import { InsertContentDetails } from './ContentTypes';
 import * as InsertList from './InsertList';
 
+const wrappedElements = [ 'pre' ];
+
+const shouldFlatten = (fragment: AstNode, parentNode: Element) => {
+  const currentNode = fragment.firstChild;
```

Comment:
This name is still confusing to me at least, can we please change it?

Reply:
I'm thinking about what name could be more clear but at the moment the only idea that I have is `shouldBeFlattened`.

something about that is just one tag pasted in the same tag like: `isPastingInTheSameTag` would lose the information that only certain tags should be flattened

something like: `shouldBeUnwapped` seems less correct to me since the unwrapping is used the way we use to just copy the content and it is not what we want to achieve.

Maybe `shouldPastContentOnly`?

Reply:
I wasn't talking about that above... this is about the `currentNode` name.

Reply:
That being said `shouldPasteContentOnly` is probably a better name for this function 👍

Reply:
ok sorry I misunderstood :sweat_smile: , so I'll also rename the function, about the name of the variable, I'm thinking about: `pastingNode`, `firstNode` or removing the variable and using `fragment.firstChild` for the 3 occurrences

Reply:
Using a variable means it will minify better so I think we should keep that. My preference would just be `firstNode` to match `lastNode`.

Reply:
ok, done